stdlib/src/builtin/bool.mojo

```diff
@@ -17,6 +17,8 @@ These are Mojo built-ins, so you don't need to import them.
 
 from utils._visualizers import lldb_formatter_wrapping_type
 
+from collections.set import Set as _Set
```

Comment:
**Question** Do we need the `as _Set` part here?  If so, why?  Why can't we just use `Set` as is below in the functions if we do `from collections import Set`?

Reply:
Wasn't there an issue with importing things into builtin files getting exposed publicly?

Reply:
ok seems fine to just import Set